script.js

```diff
@@ -0,0 +1,72 @@
+const body = document.getElementById("body-project");
+const menu = document.createElement("div");
+menu.className = "menu-style";
+body.appendChild(menu);
+
+const icon = document.createElement("i");
+icon.className = "fas fa-times";
+menu.appendChild(icon);
+
+const menuList = document.createElement("ul");
+menuList.className = "menu-list";
+menu.appendChild(menuList);
+
+const portfolio = document.createElement("li");
+portfolio.className = "portfolio";
+const portfolioLink = document.createElement("a");
+portfolioLink.className = "mobile-link";
+portfolioLink.textContent = "Portfolio";
+portfolioLink.href = "#portfolio";
+portfolio.appendChild(portfolioLink);
+menuList.appendChild(portfolio);
```

Comment:
- [ ] Good job working on the mobile menu, the general concepts has been met, but there's a catch.
- Upon running your project on the live server it shows up with the mobile menu already open, same thing goes for the Desktop screens. Kindly fix this.
  See the images below.
_Mobile screens_
![image](https://user-images.githubusercontent.com/49621279/123783589-63b82b80-d88b-11eb-86c5-31bd8598b5a5.png)
_Desktop screens_
![image](https://user-images.githubusercontent.com/49621279/123783658-76326500-d88b-11eb-87ed-d694ea29164f.png)

Reply:
thank you, just fixed it.